rebar.lock

```diff
@@ -35,6 +35,10 @@
        {ref,"d82ff25baf393eb11712721fb1b881b4b9f80176"}},
   0},
  {<<"eini">>,{pkg,<<"eini">>,<<"1.2.4">>},1},
+ {<<"eodbc">>,
+  {git,"git://github.com/arcusfelis/eodbc.git",
+       {ref,"bca45cfd8138d20c2ea600a4a375d4d7bd813527"}},
```

Comment:
eodbc doesn't compile on Mac OSX High Sierra because rebar still fetches the old version as set in the `rebar.lock` file. `rebare3 upgrade eodbc` solved the issue for me. @arcusfelis please commit change the the `rebar.lock` file.

Reply:
updated